Tests/PerformanceTest/XCTestCase+InstructionsCount.swift

```diff
@@ -0,0 +1,74 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import _InstructionCounter
+
+extension XCTestCase {
+  private var baselineURL: URL {
+    if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == "1" {
+      return URL(fileURLWithPath: #file)
+        .deletingLastPathComponent()
+        .appendingPathComponent("ci-baselines.json")
+    } else {
+      return URL(fileURLWithPath: #file)
+        .deletingLastPathComponent()
+        .appendingPathComponent("baselines.json")
+    }
```

Comment:
I meant that this could be

```suggestion
    if let baselineFile = ProcessInfo.processInfo.environment["BASELINE_FILE"] {
      return URL(fileURLWithPath: baselineFile)
    } else {
      return URL(fileURLWithPath: #file)
        .deletingLastPathComponent()
        .appendingPathComponent("baselines.json")
    }
```

And then `build-script.py` could set `BASELINE_FILE=whatever/ci-baselines.json`.